### DISCUSSION TITLE
GRPC error :HTTP/2 error code: NO_ERROR Received Goaway

### DISCUSSION THREAD

**Discussion by zhangxinars** [Kong/kong#11415] [Help]
When I use the kong gateway to proxy the grpc service, I will occasionally get this error.

HTTP/2 error code: NO_ERROR
Received Goaway

I tried to modify this parameter（http2_max_concurrent_streams 1000;）but it didn't improve.

**Comment by hanshuebner** [comment]
Hello @zhangxinars,

except for the error message, which seems to be incorrectly logged as an error (as indicated by the `NO_ERROR`), do you see any other issues?

Thanks,
Hans

**Reply by zhangxinars** [reply]
Thank you for your reply.  This error is returned when the grpc is called, although occasionally, when this error is returned, we do not get the expected return value

**Reply by zhangxinars** [reply]
Are there any restrictions?

**Reply by oowl** [reply]
Could you give us a minimal reproduce method case?

**Reply by oowl** [reply]
If we confirm this is an issue, it should be discussed in the issues section, it's not suitable to discuss this in the discussion section.
